docs/guides/simbrief.md

```diff
@@ -11,4 +11,49 @@ This guide is also available in [video form](https://youtu.be/cSY-r83pXio) on ou
 
 :::info
 This guide assumes you already have a Navigraph account and know how to set up a SimBrief flight plan. If you do not, our [video](https://youtu.be/cSY-r83pXio) covers this in detail.
-:::
\ No newline at end of file
+:::
+
+## Finding Your User ID
+Before you can start importing your SimBrief flight plans into the aircraft, you must first know what your Simbrief User ID is. This allows the aircraft, and your flightplan, to seemlessly synchronise with each other. 
+
+This six-digit ID can be found beneath your specified cruising altitude **BEFORE** generating your flight, as shown below:
+
+![simbriefid](../assets/simbrief/PilotID.png)
+
+## Intergrating SimBrief in the aircraft
+Now that you know your Simbrief User ID, you can now use this to intergrate Simbrief with the Salty 747. 
+
+From the `MENU` page in the FMC, navigate to the `SALTY` page by clicking on the corresponding LSK. 
+
+![SALTY](../assets/simbrief/SALTY.png)
+
+Next, navigate to the `SIMBRIEF` page by clicking on the corresponding LSK
+
+![SIMBRIEF](../assets/simbrief/SIMBRIEF.png)
+
+Finally, enter your Simbrief User ID (as found earlier in this section) by inputting the six-digit number using the keypad.
+
+![ID](../assets/simbrief/ID.png)
+
+## Importing a Flightplan
+Finally, you can import your Simbrief flightplan into the aircraft!
+
+:::info
+For this section I am using a Simbrief flightplan from EGLL to VHHH.
+:::
+
+Firstly, navigate to the route page using the keypad keys beneath the FMC.
+
+Then, click on `ROUTE REQUEST` to draw your Simbrief flightplan into the aircraft
+
+![ROUTEPAGE](../assets/simbrief/ROUTEPAGE.png)
+
+Wait a moment for the flightplan to be loaded, and you will be greeted with the option to `LOAD` the flightplan. I would advise clicking on the `LEGS` keypad key beneath the FMC and use the Next Page function to check for errors. If there are no errors, click `LOAD` as shown below.
+
+![LOAD](../assets/simbrief/LOAD.png)
+
+Once this is complete, you will be greeted with a screen like this:
+
+![LEGS](../assets/simbrief/LEGS.png)
+
+If you are happy, click on the green-illuminated `EXEC` button beneath the FMC to confirm the import.
```

Comment:
Maybe mention that you should wait until your whole route is imported before executing (and that this can take a while)